rebuild/src/functions.cc

```diff
@@ -0,0 +1,591 @@
+#include "functions.h"
+
+namespace {
+Napi::Value AggregatePublicKeys(const Napi::CallbackInfo &info) {
+    BLST_TS_FUNCTION_PREAMBLE
```

Comment:
```suggestion
    BLST_TS_FUNCTION_PREAMBLE(info)
```
can we do this everywhere?

Reply:
Sure!